django/http/multipartparser.py

```diff
@@ -127,11 +129,13 @@ def parse(self):
         # See if any of the handlers take care of the parsing.
         # This allows overriding everything if need be.
         for handler in handlers:
-            result = handler.handle_raw_input(self._input_data,
-                                              self._meta,
-                                              self._content_length,
-                                              self._boundary,
-                                              encoding)
+            result = handler.handle_raw_input(
+                self._input_data,
+                self._meta,
+                self._content_length,
+                self._boundary,
+                encoding
```

Comment:
include a trailing comma so if more args are added later we don't have to modify this line again